rustup-init.sh

```diff
@@ -80,6 +80,21 @@ main() {
     local _arch="$RETVAL"
     assert_nz "$_arch" "arch"
 
+    local _default_host=""
+    case "$_arch" in
+        *windows*)
+            _default_host="--default-host $_arch"
+            for arg in "$@"; do
```

Comment:
This is not necessary either, IIRC according to the GNU coreutils convention, if we pass `--default-host=A --default-host=B`, then only `B` will be recognized. So just putting the `$_default_host_override` before the rest of the args should be fine.